src/transactions/transaction.controller.ts

```diff
@@ -188,9 +188,10 @@ export class TransactionController {
       ? new PublicKey(query.receiptAddress)
       : undefined;
     const nftAddress = query.nftAddress ?? undefined;
+    const assetAddress = query.assetAddress ?? undefined;
```

Comment:
use it maybe like: 
const assetAddress = query.nftAddress ?? query.assetAddress; 
thoughts?